src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/mha.cpp

```diff
@@ -43,294 +43,314 @@ static inline std::vector<std::vector<element::Type>> precision_bf16(size_t coun
     return prc;
 }
 
-static std::map<std::string, std::string> enable_callback() {
-    return std::map<std::string, std::string>{
+static ov::AnyMap enable_callback() {
+    return ov::AnyMap({
```

Comment:
```suggestion
    return {ov::intel_cpu::snippets_mode(ov::intel_cpu::SnippetsMode::ENABLE)};
```

Plus below header file:
`#include "internal_properties.hpp"`